packages/bisheng/src/utils/source-data.js

```diff
@@ -48,10 +48,12 @@ function getPropPath(filename, sources) {
 
 function filesToTreeStructure(files, sources) {
   const cleanedSources = sources.map(source => source.replace(/^\.?\//, ''));
-  return files.reduce((filesTree, filename) => {
+  const filesTree = files.reduce((filesTree, filename) => {
     const propLens = R.lensPath(getPropPath(filename, cleanedSources));
     return R.set(propLens, filename, filesTree);
   }, {});
+  // fix: windows platform
+  return filesTree[""] || filesTree
```

Comment:
This issue take me 2 hours... 

**windows:**

![win20170324155927](https://cloud.githubusercontent.com/assets/433259/24286883/22b515ee-10b2-11e7-8e9d-cab65edf8d40.png)
**Macos:**

<img width="963" alt="screen shot 2017-03-24 at 15 00 32" src="https://cloud.githubusercontent.com/assets/433259/24286899/2fb332f8-10b2-11e7-85ae-c7c4bf6d2794.png">

Could you tell me whats the difference ?